dev_guide/deployments.adoc

```diff
@@ -352,3 +352,16 @@ Rollbacks revert an application back to a previous deployment and can be
 performed using the REST API or the CLI. See the
 link:../cli_reference/basic_cli_operations.html#deployment-operations[CLI
 Reference] for more details.
+
+== Fine-Grained Control
+In addition to rollbacks, you can can exercise fine-grained control over
+the number of replicas by using the `osc resize` command.
+For example, the following command sets the replicas in the deployment
+configuration `widespread` to 3.
+
+----
+$ osc resize dc widespread --replicas=3
+----
+
+The number of replicas will also propagate to the desired state and,
+eventually, current state of the `widespread` deployment.
```

Comment:
That deployment may not be named `widespread` and most probably isn't. I would suggest changing this to *of the deployment configured by `widespread`* or something along these lines.